pyproject.toml

```diff
@@ -40,7 +41,7 @@ requires = [
 
 [tool.hatch.envs.default]
 type = "pip-compile"
-lock-filename = "requirements/requirements-lock.txt"
+#lock-filename = "requirements/requirements-lock.txt"
```

Comment:
Had to do this to workaround https://github.com/wheel-next/wheel-stub/issues/11, so I guess I might need a better alternative before this PR can be merged.